src/components/studentCard/style.css

```diff
@@ -0,0 +1,11 @@
+@import url(../../styles/_globals.css);
+
+.personCard {
+    padding: 0;
+}
+
+.edit-icon {
```

Comment:
I see this class in teacherCard.js also, again as before remember it is applied across the whole project. If this `.edit-icon` is global but needs modifying here, it should be something like `.personCard .edit-icon {...}`